x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -104,6 +104,12 @@ jest.mock('react-router-dom', () => ({
     pathname: '/triggersActions/rules/',
   }),
 }));
+jest.mock('@kbn/alerts-ui-shared', () => ({
+  MaintenanceWindowCallout: jest.fn(() => (
```

Comment:
While the current test checks for the object’s import, it might miss potential component changes that could lead to issues. Perhaps it would be beneficial to check for the actual callOut component text, “Rule notifications are stopped while the maintenance window is running,” since that’s what we expect our users to see during a maintenance window. Additionally, we might want to ensure this message doesn’t appear when there’s no maintenance window active. What do you think?